### ISSUE TITLE
Suggestion to update technique G1 for criterion 2.4.1

### ISSUE THREAD

**Issue by HelleJuhl** [w3c/wcag#2839]
Under sufficient techniques for [criterion 2.4.1](https://www.w3.org/WAI/WCAG22/Understanding/bypass-blocks.html), you'll find the technique "[G1: Adding a link at the top of each page that goes directly to the main content area](https://www.w3.org/WAI/WCAG22/Techniques/general/G1)". Step 1 in this technique's procedure is "Check that a link is the first focusable control on the Web page." and the expected result is "All checks above are true."

However, if a web page displays a cookie message (or similar important message), I believe that focusable elements in this message should have a higher priority than the link to the main content.

I suggest that the technique is updated to address the situation mentioned above.

**Comment by alastc** [comment]
Hi @HelleJuhl,

Techniques are examples, they are known ways to pass. They are not intended to encompass all possibilities. As the [Understanding Techniques for WCAG Success Criteria](https://www.w3.org/WAI/WCAG22/Understanding/understanding-techniques#sufficient-techniques) says:
> If web content implements the sufficient techniques for a given criterion correctly and it is accessibility-supported for the content's users, it conforms to that success criterion. (**The converse is not true**; if content does not implement these sufficient techniques, it does not necessarily fail the success criteria...

In this case, the technique is saying to put the link first, but that is not required by the SC.

Overall, I think it would be better to keep the technique simple rather than trying to include more variations.

**Comment by detlevhfischer** [comment]
Also, who knows if the cookie dialog nuisance will still be around in a couple of years...

@HelleJuhl are you happy with the resoonse? can we close this issue?

**Comment by HelleJuhl** [comment]
Yes, you can close it. :) The reason for my request was due to an error I got from an automated checker, where they pointed out that the first focusable control should be a link to the main content area. This error would appear more or less whenever I checked a site with a cookie message and the only reference I could find to "first focusable control" was in that technique. Instead, it seems like the checker shouldn't have given me an error in those cases. Thanks for clarifying!
